frontend/src/app/page.tsx

```diff
@@ -1,103 +1,166 @@
+'use client'
+
 import Image from "next/image";
+import {useState} from "react";
 
 export default function Home() {
+    const [notes, setNotes] = useState('');
+    const [summary, setSummary] = useState('');
+    const [isLoading, setisLoading] = useState(false);
+    const [activeTab, setActiveTab] = useState('upload');
+
+
+    const handleNotesSubmit = async () => {
+        if(!notes.trim()) return;
+
+        setActiveTab("summary");
+        setisLoading(true);
+
+        // AI Processing link here
+    }
+    const handleReset = () =>{
+        setNotes('');
+        setSummary('');
+        setActiveTab('upload');
+
+    }
+
   return (
     <div className="font-sans grid grid-rows-[20px_1fr_20px] items-center justify-items-center min-h-screen p-8 pb-20 gap-16 sm:p-20">
```

Comment:
```suggestion
    <div className="font-sans items-center justify-items-center min-h-screen p-8 pb-20 gap-16 sm:p-20">
```
It seems like the grid makes a lot of space between the elements which makes it hard to see everything without scrolling.

If we remove it it fits perfect on the page